docs/troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+You might get the following error when installing with Homebrew:
```

Comment:
We end up with some code duplication between each of these individual sections and the main README.rst.

I think the reason I used an `rst` was because pypi used to not support markdown very well. I'm pretty sure it has good support now. We can convert the README.rst to markdown, but I wonder if we can ever get rid of the documentation duplication?

Reply:
We can make the new `README.md` point to the documentation, much like [this README](https://github.com/pallets/werkzeug)